ndmg/stats/qa_tensor.py

```diff
@@ -33,86 +34,167 @@
 import sys
 import matplotlib
 
+from dipy.viz import window, actor
+from fury.actor import orient2rgb
+
 matplotlib.use("Agg")  # very important above pyplot import
 import matplotlib.pyplot as plt
 
-
-def tensor2fa(tensors, tensor_name, dti, derivdir, qcdir):
+def generate_3_d_directions(peak_dirs,peak_values):
     """
-    outdir: location of output directory.
-    fname: name of output fa map file. default is none (name created based on
-    input file)
+    Generates 3-d data required for plotting directions for the entire brain volume
+    
+    Parameters
+    -----------
+    peak_dirs: np array of peak_dirs 
+    peak_values: np array of peak_values
+    slices: dictionary of lists of slices in x,y,z plane
+    
+    Returns
+    -----------
+    centers: np array of center points
+    directions: np array of vector directions
+    directions_colors: np array of RGB colors 
+    heights: np array of vector magnitudes
```

Comment:
docstring is a bit off here.
should be:
```
argname : type
    arg description
```

That being said, it saves space doing it this way, so I'm fine keeping it if you want, and matters only for consistency

Reply:
If you decide to leave it (which I'm fine with), feel free to just resolve this after reading.